Userland/Games/Snake/main.cpp

```diff
@@ -53,6 +53,21 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     TRY(game_menu->try_add_action(GUI::Action::create("&New Game", { Mod_None, Key_F2 }, TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/reload.png"sv)), [&](auto&) {
         game->reset();
     })));
+    static DeprecatedString const pause_text = "&Pause Game"sv;
+    auto const pause_icon = TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/pause.png"sv));
+    static DeprecatedString const continue_text = "&Continue Game"sv;
+    auto const continue_icon = TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/play.png"sv));
+    TRY(game_menu->try_add_action(GUI::Action::create(pause_text, { Mod_None, Key_Space }, pause_icon, [&](auto& action) {
```

Comment:
Hmm so the spacebar pausing isn't actually working, but I think that's an existing issue. We're currently dropping all key events on the floor that aren't an arrow or WASD key. So even the F2 button to start a new game doesn't work.

If you want a free commit, this fixes it and let's the spacebar work:
```diff
diff --git a/Userland/Games/Snake/SnakeGame.cpp b/Userland/Games/Snake/SnakeGame.cpp
index af6a2e2e3b..2e43a9d531 100644
--- a/Userland/Games/Snake/SnakeGame.cpp
+++ b/Userland/Games/Snake/SnakeGame.cpp
@@ -212,6 +212,7 @@ void SnakeGame::keydown_event(GUI::KeyEvent& event)
         queue_velocity(1, 0);
         break;
     default:
+        event.ignore();
         break;
     }
 }
```

Reply:
Oops, ty for noticing!